subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -19,6 +19,14 @@ zephyr_library_sources_ifdef(
   )
 
 if(CONFIG_BT_LL_SW_SPLIT)
+  if(CONFIG_BT_CTLR_ADVANCED_FEATURES)
+    message(WARNING "\nCONFIG_BT_CTLR_ADVANCED_FEATURES=y, Advanced Features "
+		    "in Zephyr Bluetooth Controller are EXPERIMENTAL.")
```

Comment:
I don't agree the all features under CONFIG_BT_CTLR_ADVANCED_FEATURES are EXPERIMENTAL, some are but others are just "advanced".

Reply:
Agree. But `CONFIG_BT_CTLR_ADVANCED_FEATURES` enables (makes advanced features visible) changing the Kconfig options' defaults which could lead to modification to controller's design.

Should I reword the warning?  "Changing the advanced features' default value changes Controller design."

Reply:
Yes the reword is better, but i'm still not sure what this is supposed to fix? A warning going by in the compilation wall of text doesn't prevent anybody from building a product with non-default advanced feature values.

When you say design, i assume you are referering to the qualified design listing.

I think at least every entry under CONFIG_BT_CTLR_ADVANCED_FEATURES should have a classification if the enabling/disabling of that feature changes the intended design.

Reply:
This is not fixing, but, on similar lines to the warning on `CONFIG_ASSERT`, to convey that there is changes to defaults under the advanced features which otherwise should be a conscious decision.

Reply:
@thoh-ot please review the change.